### ISSUE TITLE
1.20.1 - No way to configure tree spawn rate

### ISSUE THREAD

**Issue by Peach774** [MatrexsVigil/phc2trees#53]
The config appears to give no option to configure the spawn rate - also it spawns a config in the config folder and the server config and I don't know which is the right one

**Comment by Ezoteric** [comment]
Yes please, could you add a spacing/spawnrate config cause its way too much for liking.

In the picture these are all Pam trees, the other stuff is high grass from bop :P

![image_2024-03-27_005641828](https://github.com/MatrexsVigil/phc2trees/assets/14137626/6767f57b-41ac-475e-aeb9-ada80933c711)
